doc/releases/release-notes-3.1.rst

```diff
@@ -225,12 +235,17 @@ Boards & SoC Support
 
 * Added support for these following shields:
 
+  * Keyestudio CAN-BUS Shield (KS0411)
+  * MikroElektronika WIFI and BLE Shield
+  * X-NUCLEO-53L0A1 ranging and gesture detection sensor expansion board
 
 Drivers and Sensors
 *******************
 
 * ADC
 
+  * STM32: Added :c:macro:`ref_internal` support
```

Comment:
this macro doesn't exist

Reply:
I think it's better to do here:

```
Added support for :c:enumerator:`adc_reference:ADC_REF_INTERNAL`.
```

We do not document publicly API ops now (ADC needs to be fixed to mark that as internal indeed)

Reply:
ok, thanks will do